classes/TerrainFeatures.py

```diff
@@ -1,35 +1,35 @@
-""" The module descripes Terrain features, which are generators used to generate small structures """
+""" The module describes Terrain features, which are generators used to generate small structures """
 
-from classes import BasicClasses
-from classes.materials import Material
+from .BasicClasses import Block, Chunk, Region
+from .materials import Material
 from random import randint, choice
 
 
-def _generate_block_unsafely(chunk, chunkpos: [int, int, int], pos: [int, int, int], material: Material) -> bool:
-    """Generated a block in a given chunk unsafely -> ignores any errors """
+def _generate_block_unsafely(chunk: Chunk, chunkpos: [int, int, int], pos: [int, int, int], material: Material) -> bool:
+    """ Generates a block in a given chunk unsafely -> ignores any errors """
     try:
-        chunk.addNewBlock(chunkpos[0], chunkpos[1], chunkpos[2], BasicClasses.Block(pos[0], pos[1], pos[2], material, None))
+        chunk.addNewBlock(chunkpos[0], chunkpos[1], chunkpos[2], Block(pos[0], pos[1], pos[2], material, None))
```

Comment:
Coroutine needs to be `await`ed

Reply:
I know that, however I'm not knowledgeable enough to be able to use async statements (as such I'd better just stick to a pure sync structure)